nni/retiarii/codegen/pytorch.py

```diff
@@ -149,6 +164,11 @@ def graph_to_pytorch_model(graph_name: str, graph: Graph, placement=None) -> str
                 else:
                     node_codes.append(node_code)
 
+                # Map to module hierarchies in original search space python code
+                node_python_mappings[py_variable_name] = node.python_name
+
+    node_codes.append(f'self.{STATE_DICT_PY_MAPPING} = {node_python_mappings}')
```

Comment:
there might be large mapping in the generated code?

Reply:
I don't think so. Each node only appears once in the mapping.